src/OSPSuite.Presentation/Services/DataImporter.cs

```diff
@@ -29,57 +29,6 @@ IApplicationController applicationController
          _applicationController = applicationController;
       }
 
-      public override IList<MetaDataCategory> DefaultMetaDataCategories()
```

Comment:
@abdelr @georgeDaskalakis Not sure why this was not promoted to the base class .. This is the exact same code for both R and Presentation implementation